lib/clearLocalCache/index.native.js

```diff
@@ -0,0 +1,5 @@
+import RNFetchBlob from 'rn-fetch-blob';
+
+export default function clearLocalCache() {
+  return RNFetchBlob.session().dispose();
```

Comment:
Deleting any cached file by `rn-fetch-blob` as stated in its documentation https://github.com/joltup/rn-fetch-blob#user-content-cache-file-management